application/laravel/Dockerfile

```
@@ -0,0 +1,22 @@
+FROM composer:1.10.10
+
+# Create Laravel project
+RUN composer create-project laravel/laravel demo
+
+# Work with the Laravel Project
+WORKDIR ./demo
+
+# Install SecretHub
+RUN apk add --repository https://alpine.secrethub.io/alpine/edge/main --allow-untrusted secrethub-cli
+
+# Copy prerequisite files
+COPY welcome.blade.php ./resources/views/welcome.blade.php
```

Let's put the `welcome.blade.php` in `resources/views/` from the start, so then we can just do:

```suggestion
COPY . .
```